components/performances/PerformancesTable.js

```diff
@@ -47,7 +23,7 @@ export default function PerformancesTable({
       filter: 'matchEnum',
     },
     {
-      Header: 'Level',
+      Header: 'Perf. Lvl',
```

Comment:
Were these abbreviations part of the design?

Reply:
Yup